shell/platform/windows/key_event_handler.cc

```diff
@@ -190,6 +192,7 @@ void KeyEventHandler::HandleResponse(bool handled,
       std::cerr << "Unable to find event " << id << " in pending events queue.";
       return;
     }
+#ifndef WINUWP
```

Comment:
Add an explanatory comment/todo here referring to the null delegate.

Reply:
Fixed in next commit.